TinyConstraints/Classes/TinyConstraints+superview.swift

```diff
@@ -25,7 +25,7 @@
 #if os(OSX)
     import AppKit
     
-    public extension TinyView {
+@MainActor public extension TinyView {
```

Comment:
Not needed since `TinyView` is already isolated to the `@MainActor`

```suggestion
public extension Constraint {
```